test/general/openpgp.js

```diff
@@ -1821,6 +1846,24 @@ aOU=
       })).to.be.rejectedWith(/No signing keys provided/);
     });
 
+    it('Signing with key which uses sha3 should generate a valid sha3 signature', async function() {
+      const privKey = await openpgp.readKey({ armoredKey: priv_key_sha3 });
+      const pubKey = privKey.toPublic();
+      const text = 'Hello, world.';
+      const message = await openpgp.createCleartextMessage({ text });
+      const config = { minRSABits: 1024 };
```

Comment:
```suggestion
```

This is only necessary when using weak RSA keys, which some other tests do, but `priv_key_sha3` is fine 🙂